components/support/nimbus-fml/fixtures/ios/tests/FeatureManifest.swift

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+public func add(a: Int, b: Int) -> Int {
```

Comment:
this is to smoke test the compilation + running of scripts, this has nothing to do with the FML itself